mobly/controllers/android_device_lib/snippet_client_v2.py

```diff
@@ -109,12 +128,14 @@ class SnippetClientV2(client_base.ClientBase):
       the connection to the server is made successfully.
   """
 
-  def __init__(self, package, ad):
+  def __init__(self, package, ad, configs=None):
     """Initializes the instance of Snippet Client V2.
 
     Args:
       package: str, see base class.
       ad: AndroidDevice, the android device object associated with this client.
+      configs: Config, the configuration object. See the docstring of the
```

Comment:
is this `configs` or `config`?
Seems like it's singular?

I see we already have `def load_config(self, config):`

Same for all the other places

Reply:
I used `configs` because the config arg of all services are named in plural form, i.e. BaseService's constructor is `__init__(..., configs)`.

So looks like there's a conflct in existing config arg names. Based on this, what name would you suggest to use here? We can insist on

Reply:
I took a look in the codebase.
Let's use singular here.

Reply:
Done.